server/app/views/admin/programs/ProgramBlocksView.java

```diff
@@ -931,13 +931,7 @@ private DivTag renderReadOnlyLabel(String label) {
     DivTag ret =
         div()
             .withClasses(
-                "flex",
-                "gap-2",
-                "items-center",
-                "text-gray-400",
-                "font-medium",
-                "bg-transparent",
-                "rounded-full")
+                "flex", "gap-2", "items-center", "text-gray-400", "font-medium", "rounded-full")
```

Comment:
Why: The new ordering in the css file causes `bg-transparent` to load later now upsetting the visuals. This will be the same reason as the other `bg-transparent` removals.